apiserver/apiserver.go

```diff
@@ -888,6 +890,7 @@ func (srv *Server) endpoints() ([]apihttp.Endpoint, error) {
 	resourcesMigrationUploadHandler := srv.monitoredHandler(handlersresources.NewResourceMigrationUploadHandler(
 		&migratingApplicationServiceGetter{ctxt: httpCtxt},
 		&migratingResourceServiceGetter{ctxt: httpCtxt},
+		resourcesdownload.NewDownloader(logger, resourcesdownload.DefaultFileSystem()),
```

Comment:
todo: Make the logs easier to differentiate in the new package: `logger.Child("resourcesmigrationdownload")`